src/Ball.js

```diff
@@ -80,6 +68,23 @@ export class Ball {
     }
   }
 
+  accelerateBackToHealthyLevel () {
+    if (this.vx > 0) {
+      this.vx = Math.min(SPEED, this.vx * DECELERATION_WHEN_SICK)
```

Comment:
This code has been changed from `this.vx * 4` to `this.vx * 100`.
I'm not sure this is the expected behavior. I would expect the previous factor, while 100 seems to me a scalar value.

Reply:
Fair point, at the moment SPEED would always be taken. Will roll it back

Reply:
Change moved forward to next PR.